Chat/File.cpp

```diff
@@ -29,20 +35,40 @@ bool File::SetFile(const std::string &path)
         std::getline(str_stream, name_, '\\');
     }
 
+    HANDLE file = CreateFile(path_.c_str(),             //path
```

Comment:
One more thing. Are you sure that this function will succeeded?
Please, add checking for INVALID_HANDLE_VALUE here.
Also you should add checking after every WINAPI function for right result.